### PR TITLE
Add tzdata to base-chroot

### PR DESCRIPTION
**Pull request by sgn**
@void-linux/pkg-committers

### CODE REVIEW

**Comment by the-maldridge** [comment]
context?

File: srcpkgs/tzutils/template [hunk 1]
```diff
@@ -1,16 +1,18 @@
 # Template file for 'tzutils'
 pkgname=tzutils
 version=2021a
-revision=1
+revision=3
```
Comment (ericonr) [inline srcpkgs/tzutils/template:4]: 2?

File: srcpkgs/tzdata/template [hunk 2]
```diff
@@ -2,7 +2,7 @@
 pkgname=tzdata
 version=2021a
 revision=1
-wrksrc=tzdata
+bootstrap=yes
```
Comment (ericonr) [inline srcpkgs/tzdata/template:5]: Removing `wrksrc` seems wrong to me...

**Review by ericonr — APPROVED** [review]
It would be nice to fix the `tzutils` commit to look like the `tzdata` one for the `set bootstrap=yes` part.

Do you think we should also remove `tzdata` from `checkdepends`?

**Comment by sgn** [comment]
> context?


Some packages require tzdata to testing, however, adding `tzdata` to
those checkdepends will break masterdir, since `tzdata` provides
`/usr/share/zoneinfo/UTC`, hence, it will be removed upon cleanup.

Let's add `tzdata` into `base-chroot` and remove the shenanigan in
`chroot.sh`.

File: srcpkgs/tzdata/template (hunk 2 above)
Comment (sgn) [reply srcpkgs/tzdata/template:5]: It's `create_wrksrc=yes`, so remove `wrksrc=` is fine.

**Comment by sgn** [comment]
> Do you think we should also remove `tzdata` from `checkdepends`?

Should be done, too. But not necessary.

**Comment by sgn** [comment]
I guess no-one against this proposal, merging tomorrow.

**Pull request merged**